typescript/packages/well-log-viewer/src/components/CallbackManager.tsx

```diff
@@ -3,12 +3,12 @@ import type { LogViewer } from "@equinor/videx-wellog";
 
 import type { WellLogController } from "./WellLogView";
 import type WellLogView from "./WellLogView";
-import type { WellLog } from "./WellLogTypes";
 import type { Info } from "../components/InfoTypes";
+import type { WellLogSet } from "./WellLogTypes";
 
 export class CallbackManager {
     controller: WellLogController | null;
-    welllog: () => WellLog | undefined;
+    wellLog: () => WellLogSet[] | undefined;
```

Comment:
rename to wellLogSets

You can add a welllog function calling the new one if needed.

Reply:
I think it should be more obvious that the field is a getter function, not a value, so I made replacement called `getWellLogSets`, and deprecated the `welllog` field